app/controllers/anonymous_feedback/problem_reports_controller.rb

```diff
@@ -0,0 +1,65 @@
+require "gds_api/support_api"
+
+class AnonymousFeedback::ProblemReportsController < AuthorisationController
+  def index
+    authorize! :request, :review_feedback
+
+    api_response = fetch_problem_reports
+
+    @feedback = AnonymousFeedbackPresenter.new(api_response)
+    @dates = present_date_filters(api_response)
+  end
+
+  def review
+    authorize! :request, :review_feedback
+
+    if review_params.any? && reviewed_items_successfully?
+      redirect_to anonymous_feedback_problem_reports_path(params.slice(:to_date, :from_date, :include_reviewed))
+    else
+      flash.now[:alert] = 'Something went wrong with this review'
+      render :index, status: 400
+    end
+  end
+
+  private
+
+  def fetch_problem_reports
+    AnonymousFeedbackApiResponse.new(
+      support_api.problem_reports(api_params).to_hash
+    )
+  end
+
+  def api_params
+    {
+      exclude_reviewed: !index_params[:include_reviewed],
```

Comment:
O_o

How come we have a disconnect here? I'm a little confused as to why we'd ask the user if they wanted to include reviewed or not and then invert that on the API side.

There's probably a reason, tho...